visualizer/layout/collapsed-layout.js

```diff
@@ -212,6 +257,20 @@ class CollapsedLayout {
   isCollapsible (layoutNode) {
     return layoutNode.collapsedNodes || this.isBelowThreshold(layoutNode)
   }
+  isVerticallyCollapsible (childNode, hostNode, squashNode = childNode) {
```

Comment:
I think we can simplify this signature to take in `squashNode` only (and in the context of this fn prob. best to name the param `childNode`), btw `hostNode` is not used

Reply:
Good catch! I actually looked at the `collapseHorizontally` in last PR and thought "Oh, this one is okay because it has `|| prev` everywhere" - but that was actually incorrect

Yeah that sounds right with `isPairCollapsible`. Also just noticed `belowThreshold =` line in vertical collapse is duplicating code from `isCollapsible` so we may switch to using the fn. And horizontal collapse does not consider `.collapsedNodes` - I can't remember if this is a hole in logic or deliberate

Yeah a separate PR dedicated to fixing holes in `collapse-layout` sounds right